schemas/didDocument/didDocument.json

```
@@ -0,0 +1,41 @@
+{
+    "$id": "/did-core.didDocument",
+    "type": "object",
+    "properties": {
+      "@context": {
+        "title": "@context",
+        "description": "https://w3c.github.io/did-core-registry/#context",
+        "anyOf": [
+          { "type": "string" },
+          { "type": "array",
+            "items": [
+              {
+                "type": "string",
+                "const": "https://www.w3.org/ns/did/v1"
+              }
+            ] 
```

```suggestion
            ], 
          "additionalItems": { "type": "string" }
```

We need to ensure that integers, bools, null values aren't allowed. We need a URI type quickly so we can ensure that type matching occurs appropriately. Even with this change, this isn't strict enough.